scanner/scanner.go

```diff
@@ -405,10 +405,21 @@ func scanDiv(s *Scanner) (token.Token, []byte) {
 	return token.Div, nil
 }
 
+func isSymbolPrefix(c byte) bool {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 13 of function isSymbolPrefix() is high (> 12)

<!-- Comment by [SideCI](https://sideci.com) -->